include/btservice/btservice.hpp

```diff
@@ -18,9 +19,10 @@ class btservice
 
 private:
     const uint16_t cServicePortNumber = 22;
-
+    void connectToBluetooth(QBluetoothAddress addr, QBluetoothAddress controller);
     openauto::btservice::AndroidBluetoothService androidBluetoothService_;
     openauto::btservice::AndroidBluetoothServer androidBluetoothServer_;
+    QProcess *btConnectProcess;
```

Comment:
does this need to be a class member? seems like its only used that one time

Reply:
In the case of the pi workaround, the process is never guaranteed to finish (at least until openauto exits), so I wanted to hang onto a reference to the process just in case.

Reply:
gotcha, makes sense

so theres no guarentee it connects either right? Not a big issue, just curious

Reply:
Correct - although the process will exit in that case.